web/client/utils/ogc/Filter/CQL/parser.js

```diff
@@ -5,29 +5,31 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree.
  */
-const fromWKT = () => {
-    throw new Error("WKT parsing for CQL filter not supported yet");
-}; // TODO: use wkt-parser
-const spatialOperators = {
+import {toGeoJSON} from '../../WKT';
```

Comment:
can this be changed to **toGeoJSONGeometry**

Reply:
I think it is a correct name. WKT provides only syntax for geometries, so it is implicit.

Reply:
I strongly disagree and at this point is pointless discuss further so I'm going to approve it as you did.